apps/client/src/__mocks__/address.js

```diff
@@ -1,14 +1,14 @@
 import { loader } from "graphql.macro";
-const findAddress = loader("../../components/Location/LocationFinder.graphql");
+const findAddress = loader("../components/Location/LocationFinder.graphql");
 
 export default [
   {
     request: {
       query: findAddress,
       variables: {
-        postalCode: "1055xd",
+        extraHouseNumberFull: "",
         houseNumberFull: "19c",
-        extraHouseNumberFull: "19c",
+        postalCode: "1055XD",
```

Comment:
Can you get this information from `addressProperties.fixtures.find(([_, __, types]) => types.includes(addressProperties.MUNICIPAL_MONUMENT))`? I think this would be better than having 1055xd scattered all over the place.
Maybe create a util function for this and also export the types.

Reply:
As discussed